backend/app/crud.py

```diff
@@ -52,3 +55,53 @@ def create_item(*, session: Session, item_in: ItemCreate, owner_id: uuid.UUID) -
     session.commit()
     session.refresh(db_item)
     return db_item
+
+
+
+
+def create_patient(*, session: Session,patient_create: PatientCreate, owner_id: uuid.UUID) -> Patient:
+    db_patient = Patient.model_validate(patient_create, update={"owner_id": owner_id, "created_datetime": datetime.now()})
+    session.add(db_patient)
+    session.commit()
+    session.refresh(db_patient)
+    return db_patient
+
+
+def get_caregiver_patients(*, session: Session, caregiver: User ,skip: int , limit: int  ) -> PatientsPublic:
+    count_statement = (
+            select(func.count())
+            .select_from(Patient)
+            .where(Patient.owner_id == caregiver.id)
+        )
+    count = session.exec(count_statement).one()
+    first_100_patients_statement =  (
+        select(Patient)
+        .where(Patient.owner_id == caregiver.id)
+        .offset(skip)
+        .limit(limit)
+    )
+    patients = [p for p in session.exec(statement=first_100_patients_statement).all()]
```

Comment:
If i'm not mistaken, `PatientsPublic` takes an array of `PatientPublic`, but you're passing a list a `Patient`s.
Use you class method magic thingy here to create an array of `PatientPublic`